### ISSUE TITLE
ouput wrong position when passing empty string

### ISSUE THREAD

**Issue by azu** [wooorm/mdast#13]
```js
var mdast = require('mdast');
var emptyString = "";
var ast = mdast.parse(emptyString);
console.log(JSON.stringify(ast));
/*
{
    "type": "root",
    "children": [],
    "position": {
        "start": {
            "line": 1,
            "column": 1
        }
    }
}
*/
// position.end is undeifned
```

Example : http://requirebin.com/?gist=ad3c34ef897338867009

Expected:

```
{
    "type": "root",
    "children": [],
    "position": {
        "start": {
            "line": 1,
            "column": 1
        },
        "end": {
            "line": 1,
            "column": 1
        }
    }
}
```

Actual: 

`position.end` is undeifned

**Comment by wooorm** [comment]
Thanks for letting me know! I think I already fixed this in master (https://github.com/wooorm/mdast/commit/7c45751912ab5b8a53eaa015146cda25a1c415d6#diff-9f97a9c36fbbfb0d19068d6fae3bb84eR1490). Will push a new version today!

**Comment by azu** [comment]
@wooorm Thanks! good news for me.

**Comment by wooorm** [comment]
This was fixed by 7c45751, and released in cb86b36.

**Comment by azu** [comment]
@wooorm work fine! Thanks! :+1:
